data/repos.yml

```diff
@@ -559,6 +717,16 @@
   sentry_url: false
   dashboard_url: false
 
+# Propose archiving. This looks like a 2nd line utility that _could_ be handy during migration of people/roles in govt, but hasn't been used (as far as I know) in 4 years and potentially doesn't work anymore.
+- repo_name: govuk-people-roles-migration
+
+# Propose archiving - this is a one-time prototype, no longer used AFAIK.
+# Check with Kevin.
+- repo_name: govuk-personalisation-showcase-2022
+
+# Check with the data team. Consider archiving?
+- repo_name: govuk-polling-analysis
```

Comment:
@nacnudus

Reply:
Nothing to do with me or my team, sorry.  Perhaps one for @kevindew.

Reply:
@ChetanChauhanSims should this be archived?

Reply:
Just nudging on this @ChetanChauhanSims